lib/cloudregister/registerutils.py

```diff
@@ -1912,3 +2043,14 @@ def __generate_registry_auth_token(username=None, password=None):
         username=username,
         password=password
     ).encode()).decode()
+
+
+# ----------------------------------------------------------------------------
+def __mv_file_backup(filename):
```

Comment:
I created the method to move the files in this PR, a while ago,

I can rename it to `__create_config_backup(file_name)` in a refactor, in a different PR